core/state/statedb.go

```diff
@@ -236,6 +238,10 @@ func (s *StateDB) StopPrefetcher() {
 	}
 }
 
+func (s *StateDB) EnableWitnessCollection() {
+	s.collectWitness = true
+}
```

Comment:
We have a method for starting the prefetcher. IMO we should add witness colelction as a flag to that method instead of making a new one here. This increases the API surface for no good reason.